### ISSUE TITLE
API called 'u' failed: Server returned error ENOENT

### ISSUE THREAD

**Issue by 161134050** [megous/megatools#449]
Using an account from 2013, this error doesn't always happen, seems to be in bursts, then goes back to normal.

Happened when syncing a directory of videos

![image](https://user-images.githubusercontent.com/39843740/67019307-091a8280-f12f-11e9-86ce-d7760c470d2e.png)

**Comment by megous** [comment]
That would explain that it occurs in bursts. Megatools just moves on to the next file if it can't get upload handle for the current one. The size of the file to be uploaded is passed in too, but I guess the errors don't depend on file size, just timing?

**Comment by 161134050** [comment]
Yes, I had to re-run the command a few times to get all the files in my directory uploaded, all good now
